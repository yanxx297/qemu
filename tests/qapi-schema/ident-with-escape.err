tests/qapi-schema/ident-with-escape.json:3: Expression is missing metatype
