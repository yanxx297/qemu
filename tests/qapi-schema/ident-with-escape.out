[OrderedDict([('cu006fmmand', 'u0066u006fu006FA'), ('du0061ta', OrderedDict([('u0062u0061u00721', 'u0073u0074u0072')]))])]
[]
[]
